src/steps/2.addModules/moduleConfigs.ts

```diff
@@ -386,12 +402,27 @@ export const moduleConfigs: Record<Modules, ModuleConfig> = {
       "- [ ] Auth: Configure your auth providers to the [NuxtAuthHandler](./server/api/auth/[...].ts)",
       "- [ ] Auth, optional: Enable global protection by setting `enableGlobalAppMiddleware: true` in [your nuxt.config.ts](./nuxt.config.ts). Delete the local middleware in the [protected.vue](./pages/protected.vue) page if you do"
     ],
-    htmlForIndexVue: `<p>
-      Checkout the page protected by \`nuxt-auth\` here:
-      <nuxt-link to="/protected" class="underline text-blue">
-        Click me to test the auth setup!
-      </nuxt-link>
-    </p>`
+    htmlForIndexVue: `
```

Comment:
this html is largely always the same -> can you create a function for it that ghenerates this string so that we don't replicate it so often?